### ISSUE TITLE
Wheels for Python 3.13

### ISSUE THREAD

**Issue by tueda** [SamuraiT/mecab-python3#106]
It would be great if Python 3.13 wheels could be made available on PyPI.

Here is the result of running `pip install mecab-python3` on the [python:3.13](https://hub.docker.com/_/python) Docker image:
```
root@9822535832be:/# pip install mecab-python3
Collecting mecab-python3
  Downloading mecab_python3-1.0.9.tar.gz (78 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: mecab-python3
  Building wheel for mecab-python3 (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for mecab-python3 (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      WARNING setuptools_scm.pyproject_reading toml section missing 'pyproject.toml does not contain a tool.setuptools_scm section'
      Traceback (most recent call last):
        File "/tmp/pip-build-env-d8kdiabj/normal/lib/python3.13/site-packages/setuptools_scm/_integration/pyproject_reading.py", line 36, in read_pyproject
          section = defn.get("tool", {})[tool_name]
                    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
      KeyError: 'setuptools_scm'
      running bdist_wheel
      running build
      running build_py
      creating build/lib.linux-x86_64-cpython-313/MeCab
      copying src/MeCab/cli.py -> build/lib.linux-x86_64-cpython-313/MeCab
      copying src/MeCab/__init__.py -> build/lib.linux-x86_64-cpython-313/MeCab
      running build_ext
      error: [Errno 2] No such file or directory: 'mecab-config'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for mecab-python3
Failed to build mecab-python3
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (mecab-python3)
```

**Comment by polm** [comment]
Thanks for the heads up, I'll work on this.

**Comment by polm** [comment]
This should be resolved in v1.0.10, which is building now. Let me know if there are any issues.
